libsweep/src/unix/serial.cc

```diff
@@ -26,6 +26,15 @@ static speed_t get_baud(int32_t bitrate) {
     throw error{"Only baud rate 115200 is supported at this time."};
     return -1;
   }
+
+  // translate human readable bitrate to termios bitrate
+  if (bitrate == 115200)
+  {
+    #ifdef B115200
```

Comment:
Could you clang format as well. Format file included in the main repo. If not, let me know and I'll clean it up before merging.